packages/core/src/connectors/aliyun-dm/single-send-mail.ts

```diff
@@ -7,18 +7,25 @@ import { PublicParameters, request } from '../utilities/aliyun';
 interface SingleSendMail {
   AccountName: string;
   AddressType: '0' | '1';
-  ReplyToAddress: 'true' | 'false';
-  Subject: string;
-  ToAddress: string;
   ClickTrace?: '0' | '1';
   FromAlias?: string;
   HtmlBody?: string;
+  ReplyToAddress: 'true' | 'false';
```

Comment:
Can we use boolean for this param?

Reply:
We may keep using true/false string to follow aliyun-dm API doc.
![image](https://user-images.githubusercontent.com/15182327/151645491-032b0fc0-a1a7-4aae-994d-192c6d13c89d.png)

Reply:
OK